src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1000,6 +1007,24 @@ class DocBaseViewer extends BaseViewer {
             // Add page IDs to each page after page structure is available
             this.setupPageIds();
         }
+
+        this.initThumbnails();
+    }
+
+    initThumbnails() {
+        this.thumbnailsSidebar.init({
+            totalItems: this.pdfViewer.pagesCount,
+            itemHeight: 115,
+            containerHeight: this.docEl.clientHeight,
+            marginTop: 7.5,
+            marginBottom: 7.5,
+            renderItemFn: (page) => {
+                const thumbnail = document.createElement('button');
+                thumbnail.className = 'bp-thumbnail';
+                thumbnail.textContent = `${page}`;
```

Comment:
Does this work properly in IE11?

>Unlike textContent, altering innerText in Internet Explorer (version 11 and below) removes child nodes from the element and permanently destroys all descendant text nodes.

https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Differences_from_innerText

Reply:
This is just to create the temporary thumbnails for this PR, I'll keep that in mind for when I do a PR to cut the thumbnail images